index.html

```diff
@@ -18,40 +18,39 @@
   <header class="header">
     <div class="header__top">
       <div class="header__top-top container">
-        <img src="image/logo.png" alt="" class="header__logo">
-        <div class="header__search">
+        <a href="index.html">
```

Comment:
Aca tambien le pondría un `aria-label`
```suggestion
        <a href="index.html" aria-label="Ver pagina de inicio">
```